sys-fs/bcachefs-tools/bcachefs-tools-9999.ebuild

```diff
@@ -57,18 +59,22 @@ BDEPEND="
 		)
 	')
 	$(unpacker_src_uri_depends)
-	sys-devel/clang
+	<sys-devel/clang-$((${LLVM_MAX_SLOT} + 1))
+	virtual/pkgconfig
 	virtual/rust
 "
 
+llvm_check_deps() {
+	has_version -b "sys-devel/clang:${LLVM_SLOT}"
+}
+
 python_check_deps() {
```

Comment:
You need to define your own `pkg_setup` to call both `llvm_pkg_setup` and `python-any-r1_pkg_setup` now.